worlds/alttp/docs/plando_en.md

```diff
@@ -2,7 +2,7 @@
 
 ## Configuration
 
-1. Plando features have to be enabled first, before they can be used (opt-in).
+1. Only items plando has to be enabled before it can be used (opt-in).
```

Comment:
If Only Items Plando is an option it could be worth adding quotes around it, otherwise there are a few different ways it could be read.

Reply:
I reworded the sentence a bit now

Reply:
Love it 🔥